src/ploneintranet/workspace/workspacecontainer.py

```diff
@@ -15,3 +15,12 @@ class WorkspaceContainer(Container):
     A folder to contain WorkspaceFolders
     """
     grok.implements(IWorkspaceContainer)
+
+try:
```

Comment:
Well... attachments. :-) From a user perspective, you can post a status update and attach a file to it. From a dev perspective, a file object is stored on the workspace (or workspace container) temporarily until the status update is finished, then the file is moved over and attached to the status update.